Remove incorrect TMI from PFS description (#3460)
quiltdata/quilt: docs/Catalog/Preview.md

## Advanced: Quilt Package File Server

The Quilt Catalog supports HTML and JavaScript in preview via iframes. By default,

> preview iframes do not have access to S3.

Reviewer: > By default, preview iframes do not have access to S3.

It took me a while to understand this sentence. My first thought was, "So, the previews of files in S3 are disabled?" - but that's not the case. I still don't have a good idea of how to phrase it, but I'd say something like, "By default, previews only work for individual files, and don't support HTML files with dependencies on other files in the S3 bucket".

Author: OK I now say "external files in S3" and hopefully that clarifies.

> preview iframes do not have access to S3.

Author: ```suggestion
preview iframes do not have access to external files in S3.
```

> preview iframes do not have access to external files in S3.

Author: ```suggestion
preview iframes do not have IAM permissions and are therefore unable to access private files in S3.
```

> preview iframes do not have IAM permissions and are therefore unable to access private files in S3.

Author: ```suggestion
preview iframes do not have IAM permissions and are therefore unable to access
private files in S3.
```
